src/simsopt/util/dipole_array_helper_functions.py

```diff
@@ -0,0 +1,754 @@
+"""
+This module contains the a number of useful functions for  
+optimizing dipole arrays in the SIMSOPT code.
+"""
+__all__ = ['remove_inboard_dipoles',
+           'remove_interlinking_dipoles_and_TFs',
+           'align_dipoles_with_plasma',
+           'initialize_coils',
+           'dipole_array_optimization_function',
+           'save_coil_sets',
+           'quaternion_from_axis_angle',
+           'quaternion_multiply',
+           'rotate_vector',
+           'compute_quaternion',
+           'compute_fourier_coeffs',
+           'rho_fourier',
+           'generate_even_arc_angles',
+           'generate_windowpane_array',
+           'generate_tf_array',
+           'generate_curves',
+           'rho',
+           'a_m',
+           'b_m'
+           ]
+
+import numpy as np
+
+
+def remove_inboard_dipoles(plasma_surf, base_curves, eps=-0.4):
+    """
+    Remove all the dipole coils on the inboard side. Useful if the desired plasma
+    configuration is fairly compact, so no room for dipoles on inboard side.
+
+    Args:
+        plasma_surf: Surface object
+            The plasma boundary surface.
+        base_curves:
+            The curve objects for the dipoles in the array.
+        eps: float
+            Controls how inboard a dipole can be without being removed.
```

Comment:
can the docstring reflect how is a dipole classified as inboard?

can you clarify what eps actually is?